File: DAM_2º-año_1º/Programación/UD4-Ej1/src/Ej3.java
import java.util.Scanner;

/**
 * Crea una función que calcule un número elevado a otro, usando
 * multiplicaciones sucesivas.
 * **/
public class Ej3 {
    static void numeroElevado(int n,int pot){
        int resultado = 1;
        for (int i = 0; i < pot; i++) {
            resultado = resultado*n;
        }
        System.out.println("La potencia es: " + resultado);
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Dime un número: ");
        int num1 = scanner.nextInt();

        System.out.print("Dime la potencia a la que quieres elevar el número: ");
        int potencia = scanner.nextInt();

        numeroElevado(num1,potencia);

    }

}

File: DAM_1/DAM_2º-año_1º/Lenguaje de Marcas/UD7-JS/Capitulo3/Ej3/Ej3.js
let a = 0;
let b = 1;
let resultado = [a, b];

for (let i = 2; i <= 10; i++) {
    let c = a + b;
    resultado.push(c)
    a = b;
    b = c;
}
console.log(resultado.join(","))


